intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -830,14 +828,14 @@ \subsection{Activating and de-activating modules}
 Finally, to switch between clusters you can swap the ``cluster'' modules.
 These are special modules that change what modules are available for you,
 and what cluster your jobs will be queued in.
-By default you are working on \hpcname. To switch to, e.g., raichu you need
+By default you are working on delcatty. To switch to, e.g., raichu you need
```

Comment:
should use a `default_cluster` macro instead of delcatty name. same for raichu, should macro as well